Apiki/back-end-challenge
# Juliano Firme

### **Setup**
- Checkout the branch: `git checkout juliano-firme`
- ⚠️ Run `composer install` to install the dependecies

### **How to test**
- **In the root folder**, run `php -S localhost:8000 src/index.php` to initialize the server.
- Run `composer lint` to check if there is any formatting problem.
- Run `composer test` to run the test suite.
